meta-balena-common/recipes-support/balena-config-vars/balena-config-vars/balena-config-vars

```diff
@@ -85,6 +85,11 @@ if [ -f $CONFIG_PATH ]; then
         eval "$CONFIG_PARAMS"
     fi
 
+    # Don't propagate the `null` value further to dnsmasq configuration
+    if [ "$DNS_SERVERS" = "null" ]; then
+        DNS_SERVERS=""
+    fi
+
```

Comment:
I removed this by mistake as it wasn't clear that `null` is a valid parameter value. I thought that this was trying to catch a null value returned by `jq` when the field wasn't present in `config.json` (which can't happen due to the 8.8.8.8 default).

Could you update the comment to indicate that `null` is a valid setting in config.json and that this code is not fixing the parsing of a missing parameter? Thanks.